src/ripple/core/Config.h

```diff
@@ -107,6 +107,14 @@ class Config : public BasicConfig
     */
     bool                        RUN_STANDALONE = false;
 
+    /** Determines if the server will sign a tx, given an account's secret seed.
+
+        In the past, this was allowed, but this functionality can have security
+        implications. The new default is to not allow this functionality, but
+        a config option is included to enable this.
+    */
+    bool signingEnabled_ = false;
```

Comment:
all the other configurable bools are `public`, which does make it easier to manipulate the value for testing. I'm ok if you want to keep it private, but just noting the inconsistency with the other fields.